doc/articles/features/firebase-cloud-messaging.md

```diff
@@ -0,0 +1,335 @@
+# Implementing Firebase Cloud Messaging (FCM) in Uno Platform Apps
```

Comment:
We'll need to add the UID section to the top of the file like we have in the other pages, example: https://github.com/unoplatform/uno/blob/5ddf04c9dc95f7b01b07d83082982d778ca28cea/doc/articles/contributing/guidelines/creating-tests.md?plain=1#L1-L3

Just needs to be some sort of relevant unique id for the page. Following the same pattern for other files in this directory, we should probably use something like `Uno.Features.FirebaseCloudMessaging`